### PR TITLE
Added missing closing div tag

### PR DESCRIPTION
**Pull request by NamrataChangani**
## This PR is a:
- [ ] Content fix or rewrite
 

## Summary
 Added missing closing div tag

When this pull request is merged, it will...

### CODE REVIEW

File: _includes/install/tls-repo.md [hunk 1]
```diff
@@ -19,4 +19,5 @@ The following error displays if you attempt to run `composer create-project` to
 ### Using the Web Setup Wizard
 Actions like saving your authentication credentials or synchronizing with Magento Marketplace fail with the following error:
 
-![SSL connect error]({{ site.baseurl }}/common/images/install_ssl-connect-error.png)
+    ![SSL connect error]({{ site.baseurl }}/common/images/install_ssl-connect-error.png)
```
Comment (shrielenee) [inline _includes/install/tls-repo.md]: Remove this tab over - it is not needed.

File: _includes/migration/find-version.md [hunk 2]
```diff
@@ -5,13 +5,13 @@ To find the version of the Magento software:
 
 *	If you downloaded the Magento software using a Composer metapackage, enter the following command:
 
-		php <your Magento install dir>/bin/magento --version
+	php <your Magento install dir>/bin/magento --version
```
Comment (shrielenee) [inline _includes/migration/find-version.md]: Revert this change.

File: _includes/migration/find-version.md [hunk 3]
```diff
@@ -5,13 +5,13 @@ To find the version of the Magento software:
...
 *	If you cloned the Magento 2 GitHub repository, enter the following commands:
 
-		cd <your Magento 2 clone directory>
-		git branch
+	cd <your Magento 2 clone directory>
+	git branch
```
Comment (shrielenee) [inline _includes/migration/find-version.md]: Revert this change for both lines.

File: _includes/migration/find-version.md [hunk 4]
```diff
@@ -5,13 +5,13 @@ To find the version of the Magento software:
...
 
-	If you're currently in the `develop` branch, you must change to a <a href="{{ page.baseurl }}/install-gde/install/cli/dev_downgrade.html">released branch</a> before you continue.
+*   If you're currently in the `develop` branch, you must change to a <a href="{{ page.baseurl }}/install-gde/install/cli/dev_downgrade.html">released branch</a> before you continue.
```
Comment (shrielenee) [inline _includes/migration/find-version.md]: Revert this change.

File: _includes/performance/development-environment.md [hunk 5]
```diff
@@ -39,3 +39,5 @@ If you develop on a VM and it takes longer than 2 seconds to load a Magento page
 {:target="_blank"}
 [disable the caches]: {{ page.baseurl }}/config-guide/cli/config-cli-subcommands-cache.html#config-cli-subcommands-cache-en
 {:target="_blank"}
+
```
Comment (shrielenee) [inline _includes/performance/development-environment.md]: Revert this change.

Comment (NamrataChangani) [reply _includes/performance/development-environment.md]: Ok

**Comment by NamrataChangani** [comment]
@shrielenee : Hello, 
I've reverted mentioned changes by you. Thanks :)

**Comment by shrielenee** [comment]
Thanks @NamrataChangani!
